core/capabilities/compute/compute.go

```diff
@@ -294,7 +294,6 @@ func (c *Compute) createFetcher() func(ctx context.Context, req *wasmpb.FetchReq
 		)
 
 		messageID := strings.Join([]string{
-			req.Metadata.WorkflowId,
 			req.Metadata.WorkflowExecutionId,
```

Comment:
Context: the workflow execution ID is already unique and the message ID maximum is 128, so we were exceeding it.